tools/pt/libipt/ptxed/CMakeLists.txt

```diff
@@ -30,6 +30,7 @@ set(XED_LIBDIR  "" CACHE PATH "")
 include_directories(
   include
   ../libipt/internal/include
+  ../../obj/wkit/include/xed
```

Comment:
This line is no longer needed once you fix the relative paths in `tools/pt/Makefile`

```suggestion
```